### PR TITLE
[issue_tracker] Module filter has entry [object Object]

### LINKED ISSUE #6519: [issue_tracker] Module filter has entry [object Object]
In the Issue Tracker v 23.0, the module filter contains an entry labeled [object Object]

To reproduce:
* Open the Issue Tracker
* Click on the Module filter

### PR DESCRIPTION
**Pull request by laemtl**
I updated the issue_tracker class to avoid Object object to be populated in the Module dropdown.
I used getLongName() to retrieve the name of the module and populate the json.

* Resolves #6519

### CODE REVIEW

**Comment by driusan** [comment]
@lamtl needs PHPCS

```
FILE: .../travis/build/aces/Loris/modules/issue_tracker/php/issue_tracker.class.inc

--------------------------------------------------------------------------------

FOUND 1 ERROR AFFECTING 1 LINE

--------------------------------------------------------------------------------

 120 | ERROR | [x] Equals sign not aligned with surrounding assignments;

     |       |     expected 11 spaces but found 1 space

--------------------------------------------------------------------------------

PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY

--------------------------------------------------------------------------------
```

**Comment by laemtl** [comment]
Sorry I forgot to check before my PR!

File: modules/issue_tracker/php/issue_tracker.class.inc [hunk 1]
```diff
@@ -115,7 +115,12 @@ class Issue_Tracker extends \NDB_Menu_Filter_Form
             $assignees[$a_row['UserID']] = $a_row['Real_name'];
         }
 
-        $modules = \Module::getActiveModules($db);
+        $mod_descriptors = \Module::getActiveModules($db);
```
Comment (xlecours) [inline modules/issue_tracker/php/issue_tracker.class.inc:118]: I'll be a bit picky here but the variable name is not quite accurate. The returned value of `\Module::getActiveModules` is an array of `\Module` instances. It should probably be `$modules` but I understand that the intent is to use $modules to store the results of the next foreach loop. `$active_modules` would do it.

Suggestion: You could make 2 hits one stone by using array_reduce.
```
$modules = array_reduce(\Module::getActiveModules($db), function ($carry, $module) {
    ...
}, [])
```

**Comment by ridz1208** [comment]
@h-karim can you please test it out and add the passed manual tests if everything is good

**Comment by h-karim** [comment]
LGTM: Everything works well, filter shows all the modules.

**Comment by christinerogers** [comment]
@driusan final review

**Pull request merged**